config/src/config.rs

```diff
@@ -276,6 +281,17 @@ pub struct Mining {
     pub enabled: bool,
 }
 
+/// Ntp-related configuration
```

Comment:
```suggestion
/// NTP-related configuration
```

And we should probably explain somewhere what is NTP and why it's useful.

Reply:
in witnet.io documentation?

Reply:
Ok, created https://github.com/witnet/documentation/issues/20